packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -205,6 +209,13 @@ class PerformanceController extends DisposableController
         }
       }));
 
+      // Listen for Flutter.RebuiltWidgets events.
+      autoDispose(serviceManager.service.onExtensionEvent.listen((event) {
+        if (event.extensionKind == 'Flutter.RebuiltWidgets') {
```

Comment:
can we move this code into the listener above instead of initializing another listener?